.travis.yml

```diff
@@ -11,6 +11,7 @@ before_script:
   - sudo service mysql stop
   - docker-compose version
   - docker-compose up -d
+  - docker run -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=yourStrong(!)Password' -p 14330:1433 -d mcr.microsoft.com/mssql/server:2017-CU8-ubuntu
```

Comment:
Is this a commonly used password?, why adding these parameters?

Reply:
The container will terminate itself if some of the params aren't given. e.g. password under 8 chars. I copied the params from [here](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker?view=sql-server-ver15&pivots=cs1-bash#pullandrun2019)